descope/internal/mgmt/accesskey_test.go

```diff
@@ -129,23 +131,23 @@ func TestAccessKeyUpdateSuccess(t *testing.T) {
 		require.NoError(t, helpers.ReadBody(r, &req))
 		require.Equal(t, "ak1", req["id"])
 		require.Equal(t, "abc", req["name"])
-		require.Equal(t, "desc", req["description"])
+		require.Equal(t, desc, req["description"])
 	}, response))
-	res, err := mgmt.AccessKey().Update(context.Background(), "ak1", "abc", "desc")
+	res, err := mgmt.AccessKey().Update(context.Background(), "ak1", "abc", &desc)
```

Comment:
you're missing a test where you pass a nil description and check that the `description` is not in the sent request